lib/rules/require-padding-newlines-after-blocks.js

```diff
@@ -105,6 +105,15 @@ module.exports.prototype = {
                     return;
                 }
 
+                // Check if this block is used in call expression
+                if (parentNode.parentNode.type === 'CallExpression') {
```

Comment:
According to @TheSavior comment: feels like we need to add an exception for `Program` node. No?